### PR TITLE
CR-993: Making patientInfo the default behavior

### PR DESCRIPTION
**Pull request by RyanRatcliff**
This update removes the optional addition of patientInfo to the manifest and always adds it.  The genomics monitor will still attempt to find an existing patient first, but if that fails, a new patient is created.

In the below screenshot, Fred was created on demand and Wilma was already generated.  I ran FMI ingest (using file actions to be as complete a test as possible) and Fred was created and both workflows completed with the manifests kicking off ingest.
image

TIL: Testing this took a bit longer than I expected because of how glob patterns pick up files.  Looking for the following pattern `/tmp/**/*.vcf`  caused an issue with Wilma.  Turns out, the `/**/` doesn't enforce a directory, but allows the directory to exist and looks down it.  So the task was picking up `/tmp/unsorted.vcf` instead of `/tmp/Wilma_Flinstone/Wilma_Flintstone.vcf`  So some alphabetical order work on the wildcard path.  To fix this, I just delete the `unsorted.vcf` from the container after the sorted final version is created.

### CODE REVIEW

**Comment by RyanRatcliff** [comment]
Can this get some attention so that I can update or merge please and thanks

File: src/util/ga4gh.py [hunk 1]
```diff
@@ -70,20 +70,19 @@ def get_test_yml(results_payload_dict, base_xml_name, output, source, includePat
...
-            "dob": get_date(pmi.get("DOB")),
-            "identifiers": [
-                {
-                    "codingSystem": "http://hl7.org/fhir/v2/0203",
-                    "codingCode": "MR",
-                    "value": pmi.get("MRN"),
-                }
-            ],
-        }
+    yaml_file["tests"][0]["patientInfo"] = {
```
Comment (simons5593) [inline src/util/ga4gh.py:73]: I take it that the property is _always_ populated in `yaml_file`? My fear is that it isn't and that now this could fail.

**Comment by simons5593** [comment]
> Can this get some attention so that I can update or merge please and thanks

Sorry for the delay! Other than one question, this looks good!

**Review by simons5593 — APPROVED** [review]

File: src/util/ga4gh.py (hunk 1 above)
Comment (simons5593) [reply src/util/ga4gh.py:73]: Ryan and I spoke offline. This is setting the `yaml_file` not, reading it. I need more coffee.

Comment (RyanRatcliff) [reply src/util/ga4gh.py:73]: Actually, it's the opposite, we're always populating the `yaml_file` with that data now to give `genomics-ga4gh` the best chance of success.

**Pull request merged**